### ISSUE TITLE
[Feature Request] Search/filters/fast switch in Collections

### ISSUE THREAD

**Issue by joe-highnote** [apollographql/apollo-studio-community#160]
This would be great for jumping between and finding operations and collections quickly

**Comment by cheapsteak** [comment]
thanks for taking time to leave feedback!

could you expand on what you're thinking of?

**Comment by joe-highnote** [comment]
@cheapsteak absolutely! What I am envisioning is similar to the way that doc search works with cmd +k where you can start typing the name of an operation or collection and be presented a list of matches that you can select and jump to quickly. The same would be nice for open workspaces, too!

**Comment by khanakia** [comment]
Yes, this feature is essential when you have a big collection. 

I am having the same issue I cannot filter within the collection and I have to manually scroll through and search for queries.
